interfaces/Config/templates/config_sorting.tmpl

```diff
@@ -435,6 +435,12 @@
     }
 
     \$(document).ready(function() {
+
+        \$("[name='sort_cats'], [name='sort_cats_display']").multiselect( {
+            buttonTextAlignment: 'left',
+            maxHeight: 200,
+        });
+
```

Comment:
Is there a way to give it a minum width? It's a big ugly with the different sizes depending on what is selected.

Reply:
i can, theres a few options let me see if you like this one..